src/features/events/store.ts

```diff
@@ -193,6 +193,7 @@ const eventsSlice = createSlice({
           item.data = { ...item.data, reminder_sent: new Date().toISOString() };
         }
       });
+      state.participantsByEventId[eventId].isStale = true;
```

Comment:
I see now that there is actually code (on line 191) to update the reminders directly without setting it as stale. That's nice, but it's clearly not working.

I believe that's because it's using `map()` but not assigning the return value to anything. I think an alternative solution that would not require a separate load would be this:

```tsx
state.participantsByEventId[eventId].items = state.participantsByEventId[eventId].items.map((item) => {
  if (item.data && item.data?.reminder_sent !== null) {
    item.data = { ...item.data, reminder_sent: new Date().toISOString() };
  }
});
```

Reply:
I see, I did not understand that that was what that code did. It actually worked without assigning the return value. The condition was the wrong way around, checking for already sent reminders and trying to update their dates.